.github/workflows/add-lang.yml

```diff
@@ -58,4 +58,4 @@ jobs:
         run: |
           docker run \
           -v ${{ github.workspace }}:/srv/jekyll -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
-          jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 755 /srv/jekyll && jekyll build --future"
+          jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 777 /srv/jekyll && jekyll build --future"
```

Comment:
Already changed that on this repo.

Reply:
You changed it in the jekyll file but not in add-lang

Reply:
Ok. Thanks